libs/ledger-live-common/src/families/stacks/bridge/utils/misc.ts

```diff
@@ -65,71 +66,103 @@ export const getAddress = (a: Account): Address =>
     : { address: a.freshAddress, derivationPath: a.freshAddressPath };
 
 export const mapTxToOps =
```

Comment:
Could you please add a unit test for "send-many" (and maybe standard "token_transfer" ) case ? :pray: so we keep knowledge on how it's treated

Reply:
Hey @hedi-edelbloute!! Added some unit tests and also the bridge tests snapshots now include send-many txns.

Reply:
@lawRathod perfect thanks :ok_hand: